app/scripts/modules/core/src/presentation/formsy/formsy-react.d.ts

```diff
@@ -0,0 +1,35 @@
+// https://github.com/christianalfoni/formsy-react/issues/191#issuecomment-144872142
+
+// FormComponent.ts
+// Type wrapper for Formsy-React
+// Project: https://github.com/christianalfoni/formsy-react
+
+declare module 'formsy-react' {
```

Comment:
there are no `@types/formsy-react`, so this is a very incomplete typings file for it